_data/master/navbars/reference.yml

```diff
@@ -107,6 +107,26 @@ toc:
   section:
   - title: RBAC for etcdv2
     path: /reference/advanced/etcd-rbac
+- title: "Calico's driver for OpenStack (networking-calico)"
+  section:
+  - title: networking-calico
+    path: /reference/networking-calico/
+  - title: Detailed Semantics
+    path: /reference/networking-calico/semantics
+  - title: Floating IPs
+    path: /reference/networking-calico/floating-ips
+  - title: Service IPs
+    path: /reference/networking-calico/service-ips
+  - title: DevStack
+    path: /reference/networking-calico/devstack
+  - title: Kuryr
```

Comment:
Could you make sure that the sidebar titles match the document titles?

I haven't checked them all, but noticed this one doesn't match.